### PR TITLE
Lk/integration

### PR DESCRIPTION
**Pull request by leangseu-edx**
Review by commit is recommended.

Tested with real data:
- [x] Submission
- [x] Self grade
- [x] Student Training
- [x] Graded

Not tested:
- [ ] peers
- [ ] staff
- [ ] everything in between

### CODE REVIEW

File: src/components/Assessment/EditableAssessment/index.jsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ const EditableAssessment = () => {
             input={(
               <RadioCriterion
                 criterion={criterion}
+                isGrading={true}
```
Comment (muselesscreator) [inline src/components/Assessment/EditableAssessment/index.jsx:40]: plz drop this commit.  I am *completely* overhauling these components.

File: src/components/ProgressBar/index.jsx [hunk 2]
```diff
@@ -34,33 +36,37 @@ export const stepCanRevisit = {
...
 
   return (
     <Navbar className={classNames('px-0', className)}>
-      <Navbar.Collapse className="ora-progress-nav-group bg-white">
-        <hr className="ora-progress-divider" />
-        {stepEl(stepNames.submission)}
-        {stepOrder.map(stepEl)}
-        {stepEl(stepNames.done)}
+      <Navbar.Collapse className='ora-progress-nav-group bg-white'>
+        <hr className='ora-progress-divider' />
```
Comment (muselesscreator) [inline src/components/ProgressBar/index.jsx:66]: plz fix quotes

File: src/components/ProgressBar/index.jsx [hunk 3]
```diff
@@ -34,33 +36,37 @@ export const stepCanRevisit = {
...
+        {stepOrders.map((step, index) =>
```
Comment (muselesscreator) [inline src/components/ProgressBar/index.jsx:67]: `{stepOrders.map(stepEl)}`

File: src/data/services/lms/urls.js [hunk 4]
```diff
@@ -22,10 +22,11 @@ export const useViewUrl = () => {
...
-  if ([stepNames.submission, stepNames.xblock].includes(step)) {
-    return `${baseUrl}/get_block_learner_submission_data`;
-  }
-  return `${baseUrl}/get_block_learner_assessment_data/${step}`;
+  // if ([stepNames.submission, stepNames.self, stepNames.xblock].includes(step)) {
+  //   return `${baseUrl}/get_learner_data`;
+  // }
+  // return `${baseUrl}/get_learner_data/${step}`;
+  return `${baseUrl}/get_learner_data`;
 };
```
Comment (muselesscreator) [inline src/data/services/lms/urls.js:30]: Why not passing the step?  is required for the features to work.

Comment (leangseu-edx) [reply src/data/services/lms/urls.js:30]: This probably has to be step 2. It produced more bugs on the backend than we bargain for.

Comment (muselesscreator) [reply src/data/services/lms/urls.js:30]: for the sake of immediate integration, I propose adding the step *if it is in the accepted list*.

File: src/views/StatusAlert/SelfAssessment/CompleteStatusAlert.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```
Comment (muselesscreator) [inline src/views/StatusAlert/SelfAssessment/CompleteStatusAlert.jsx:1]: I believe this whole view folder has been removed. plz integrate w/ the top-level StatusAlerts component.

**Review by muselesscreator — APPROVED** [review]
LGTM

**Pull request merged**